apps/meteor/client/views/room/components/body/composer/ComposerOmnichannel/hooks/useResumeChatOnHoldMutation.ts

```diff
@@ -1,18 +1,24 @@
 import type { IRoom } from '@rocket.chat/core-typings';
-import { useMethod } from '@rocket.chat/ui-contexts';
+import { useEndpoint, useToastMessageDispatch } from '@rocket.chat/ui-contexts';
 import type { UseMutationOptions, UseMutationResult } from '@tanstack/react-query';
 import { useMutation, useQueryClient } from '@tanstack/react-query';
 
 export const useResumeChatOnHoldMutation = (
 	options?: Omit<UseMutationOptions<void, Error, IRoom['_id']>, 'mutationFn'>,
 ): UseMutationResult<void, Error, IRoom['_id']> => {
-	const resumeChatOnHold = useMethod('livechat:resumeOnHold');
+	const resumeChatOnHold = useEndpoint('POST', '/v1/livechat/room.resumeOnHold');
+
+	const dispatchToastMessage = useToastMessageDispatch();
 
 	const queryClient = useQueryClient();
 
 	return useMutation(
-		async (rid) => {
-			await resumeChatOnHold(rid, { clientAction: true });
+		async (roomId) => {
+			try {
+				await resumeChatOnHold({ roomId });
+			} catch (error) {
+				dispatchToastMessage({ type: 'error', message: error });
```

Comment:
do not handle errors inside the mutation, there is a prop to manage that

Reply:
Thanks for the tip! I wasn't aware of this